query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseFunctionField.java

```diff
@@ -25,18 +28,47 @@
 import org.codice.ddf.admin.common.report.FunctionReportImpl;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
 
 public abstract class BaseFunctionField<T extends DataType> extends BaseField<Map<String, Object>, FunctionReport<T>> implements FunctionField<T> {
 
     private FunctionReportImpl<T> report;
 
+    private String description;
+
     public BaseFunctionField(String functionName, String description) {
         super(functionName, description);
+        this.description = description;
         report = new FunctionReportImpl<>();
     }
 
+    @Override
+    public String description() {
+        Set<String> errors = getErrorCodes();
+        if (!errors.isEmpty()) {
+            return String.format("%s %n%n The possible errors are: %n- %s",
```

Comment:
`%n-` creates a bulleted list so I don't need `\r%n` and `%n%n` tells it it's another paragraph.